bots/jabber_mirror_backend.py

```diff
@@ -298,32 +298,31 @@ def config_error(msg):
 
 if __name__ == '__main__':
     parser = optparse.OptionParser(epilog=
-'''Most general and Jabber configuration options may also be specified in the
-zulip configuration file under the jabber_mirror section (exceptions are noted
-in their help sections).  Keys have the same name as options with hyphens
-replaced with underscores.  Zulip configuration options go in the api section,
-as normal.'''.replace("\n", " ")
-                                    )
+                                   '''Most general and Jabber configuration options may also be specified in the
+                                   zulip configuration file under the jabber_mirror section (exceptions are noted
+                                   in their help sections).  Keys have the same name as options with hyphens
+                                   replaced with underscores.  Zulip configuration options go in the api section,
+                                   as normal.'''.replace("\n", " "))
```

Comment:
Strings created through triple quotes preserve whitespaces in Python e.g.
```Python
In [8]: print('''
   ...:       line 1
   ...:       line 2
   ...:       '''.replace('\n', ' '))
       line 1       line 2
```
The easiest way would be to use the fact that the Python compiler concatenates two strings together if they don't have any operator in between them e.g.
```Python
In [9]: print('line 1 '
   ...:       'line 2')
line 1 line 2
```
This comment applies to other multi-line strings as well.